invenio_rdm_records/oaiserver/services/services.py

```diff
@@ -34,10 +34,10 @@
 class OAIPMHServerService(Service):
     """OAI-PMH service."""
 
-    def __init__(self, config, reserved_prefix="community-"):
+    def __init__(self, config, extra_reserved_prefixes={}):
```

Comment:
have you tested if this is compatible with previous instantiations of the service? e.g. the ones that were passing a value in `reserved_prefix`, thus not assuming the default.

Reply:
The `reserved_prefix` was added 4 commits ago (see commit [here](https://github.com/inveniosoftware/invenio-rdm-records/commit/766c19ec785df69a845a8152f4e38b0d550d4f25) ) but the PR was merged and here I am just addressing some comments and improving it, so there are no backwards compatibility issues, but thanks for keeping this in mind while reviewing :)

The "default" `reserved_prefixes` can be customized/overridden by changing the service config, and when instantiated you can extend them, so it's quite "customizable" I would say